modules/002-deckhouse/crds/deckhouse-release.yaml

```diff
@@ -69,8 +69,10 @@ spec:
                   enum:
                     - Pending
                     - Deployed
-                    - Outdated
+                    - Outdated # deprecated
                     - Suspended
+                    - Superseded
+                    - Skipped
```

Comment:
Should we have 'Cancelled' here?

Reply:
What does it mean? `Cancelled`

Reply:
It's how we are going to cancel update, skip it intentionally. I guess, we don't have such a scenario for now.

Reply:
`Suspended` status covers this kind of a scenario